### ISSUE TITLE
ForbidEnumInFunctionArgumentsRule shouldn't report error when SORT_REGULAR is used as second argument

### ISSUE THREAD

**Issue by kamil-zacek** [shipmonk-rnd/phpstan-rules#195]
Argument 1 in array_unique() cannot contain enum as the function causes implicit __toString conversion which is not supported for enums

For this code 

```
$countries = [Country::CZ, Country::CZ, Country::SK];
array_unique($countries, SORT_REGULAR)
```
shouldn't be reported this error, because of second argument the __toString is not used for compare.

Probably there are some other combinations of other functions which check this rule where there are different default $flag for comparing method type.

**Comment by janedbal** [comment]
Released as [2.11.1](https://github.com/shipmonk-rnd/phpstan-rules/releases/tag/2.11.1)
